contracts/EnigmaSimulation.sol

```diff
@@ -486,4 +516,30 @@ contract EnigmaSimulation is EnigmaStorage, EnigmaEvents, Getters {
     {
         return WorkersImplSimulation.verifyReportImpl(_data, _signature);
     }
+
+    /**
+    * Upgrade Enigma Contract
+    * @param _updatedEnigmaContractAddress Updated newly-deployed Enigma contract address
+    */
+    function upgradeEnigmaContract(address _updatedEnigmaContractAddress)
+    public
+    onlyOwner
+    isUpdatedEnigmaContract
+    {
+        return UpgradeImpl.upgradeEnigmaContractImpl(state, _updatedEnigmaContractAddress);
+    }
+
+    /**
+    * Upgrade Enigma Contract
```

Comment:
Same here, though this will be caught by the checkContracts script

Reply:
Resolved in a6843dc838e74a9c47cbcbc5093ad66c8e702dd3